### ISSUE TITLE
Reproduce the results

### ISSUE THREAD

**Issue by redman0226** [lartpang/ZoomNet#1]
Hi, 
Thanks for sharing the code. I am interested in your paper.
I followed your instructions and ran the code with the default setting, but I cannot reproduce the results you proposed in the paper. 
The following are the results I reproduced:
CHAMELEON: 'MAE': 0.032, CAMO: 'MAE': 0.075, COD10K: 'MAE': 0.033
I am wondering how to reproduce your results.
Thank you.

**Comment by lartpang** [comment]
@redman0226 

Thank you for your attention.
To get our results, first make sure to use the same device (RTX 2080Ti) and environment (See readme.md) as ours.
In addition, in the OneCycle learning rate decay strategy we adopted, a larger learning rate (0.05) was used. Such a setting can be used for regularization, which may also introduce some fluctuations in the training process.
This is what we hope to solve in our future work.

**Comment by lvyunqiu** [comment]
Dear author,

I downloaded your code directly and also have the results CAMO: "MAE": 0.076. You mentioned that the decay strategy is OneCycle. So I guess this strategy affects the final results. Would you provide the configuration of superparameters for OneCycle please? Further,  it is mentioned that "The learning rate is initialized to 0.05 and follows a linear warm-up and linear decay strategy" in your paper.

**Comment by lvyunqiu** [comment]
> Hi, Thanks for sharing the code. I am interested in your paper. I followed your instructions and ran the code with the default setting, but I cannot reproduce the results you proposed in the paper. The following are the results I reproduced: CHAMELEON: 'MAE': 0.032, CAMO: 'MAE': 0.075, COD10K: 'MAE': 0.033 I am wondering how to reproduce your results. Thank you.

Hi, 

Have you obtained the best results reported in the paper? If yes, how to get it?

**Comment by dddraxxx** [comment]
> Hi, Thanks for sharing the code. I am interested in your paper. I followed your instructions and ran the code with the default setting, but I cannot reproduce the results you proposed in the paper. The following are the results I reproduced: CHAMELEON: 'MAE': 0.032, CAMO: 'MAE': 0.075, COD10K: 'MAE': 0.033 I am wondering how to reproduce your results. Thank you.

Same result here. I guess it is best to provide the original config in paper for us to reproduce.

**Comment by lvyunqiu** [comment]
> > Hi, Thanks for sharing the code. I am interested in your paper. I followed your instructions and ran the code with the default setting, but I cannot reproduce the results you proposed in the paper. The following are the results I reproduced: CHAMELEON: 'MAE': 0.032, CAMO: 'MAE': 0.075, COD10K: 'MAE': 0.033 I am wondering how to reproduce your results. Thank you.
> 
> Same result here. I guess it is best to provide the original config in paper for us to reproduce.

Hello，

I have obtained the best results in the paper. I delete my pre-trained resnet50-19c8e357.pth and then follow the README again.

**Comment by dddraxxx** [comment]
> > > Hi, Thanks for sharing the code. I am interested in your paper. I followed your instructions and ran the code with the default setting, but I cannot reproduce the results you proposed in the paper. The following are the results I reproduced: CHAMELEON: 'MAE': 0.032, CAMO: 'MAE': 0.075, COD10K: 'MAE': 0.033 I am wondering how to reproduce your results. Thank you.
> > 
> > 
> > Same result here. I guess it is best to provide the original config in paper for us to reproduce.
> 
> Hello，
> 
> I have obtained the best results in the paper. I delete my pre-trained resnet50-19c8e357.pth and then follow the README again.

I see. Thanks so much for your answer.

**Comment by lartpang** [comment]
Thanks for your comments and I am closing this issue now.
Please feel free to reopen it if you still have questions, or open an new issue.
